src/redux/members/thunks.js

```diff
@@ -1,12 +1,135 @@
-import * as actions from './actions';
+import {
+  resetInitialState,
+  getMembersPending,
+  getMembersSuccess,
+  getMembersError,
+  getMemberByIdPending,
+  getMemberByIdSuccess,
+  getMemberByIdError,
+  updateMemberPending,
+  updateMemberSuccess,
+  updateMemberError,
+  createMemberPending,
+  createMemberSuccess,
+  createMemberError,
+  deleteMemberPending,
+  deleteMemberSuccess,
+  deleteMemberError
+} from './actions';
+
+export const getMembers = () => {
+  return async (dispatch) => {
+    try {
+      dispatch(getMembersPending());
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/members`);
+      const data = await response.json();
+      if (data.error) {
+        throw new Error(data.message);
+      }
+      dispatch(getMembersSuccess(data.data));
+    } catch (error) {
+      dispatch(getMembersError(error));
+    }
+  };
+};
+
+export const getMemberById = (id) => {
```

Comment:
You can't use a getById, you have to search for the element only using the getAll endpoint

Reply:
Removed